### DISCUSSION TITLE
Disable default peerjs TURN server

### DISCUSSION THREAD

**Discussion by SujithManjavana** [peers/peerjs#1034] [Q&A]
Is there any way to disable the default peerjs cloud TURN server and use only the custom TURN server? How to know which TURN server is currently being used?

**Comment by jonasgloning** [comment]
Setting a custom TURN server _replaces_ the default servers.

You can set `debug: 3` and look at the ICE candidates. They should contain the TURN server address.

**Reply by SujithManjavana** [reply]
will it use the default server if my custom TURN server is down?

**Reply by jonasgloning** [reply]
No.
If you want that, you can extend the default config with your servers, and the client will pick the fastest available server. However, too many servers can result in slower connection establishment because all of them will be contacted.
